crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -195,6 +225,23 @@ impl<'w, 's, 'a> BuildChildren for EntityCommands<'w, 's, 'a> {
         self
     }
 
+    fn add_children<T>(&mut self, spawn_children: impl FnOnce(&mut ChildBuilder) -> T) -> T {
+        let parent = self.id();
+        let (out, push_children) = {
```

Comment:
Not a huge fan of the code quality of the 'block returning tuple' pattern.

I see that it's to make the lifetimes of self.commands() work out, but I'm surprised that nll doesn't see that `commands` is never used after the call and end the lifetime.

Reply:
Yup, looks like you're right. Fixed.